setup.py

```diff
@@ -75,15 +75,15 @@ def is_requirement(line):
     ],
     include_package_data=True,
     install_requires=load_requirements('requirements/base.in'),
-    license="AGPL 3.0",
+    license="Apache",
```

Comment:
`"Apache 2.0"` I think?

Reply:
no idea.  is not included in the codejail setup.py.  Happy to update to this if that is your preference.